CHANGELOG.md

```diff
@@ -0,0 +1,37 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/), and this project
+tries to adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+
+## [Unreleased]
+
+### Added
+
+- Started keeping a changelog. Added a changelog file, and added a changelog nagger to CI.
+
+### Changed
+
+### Deprecated
+
+### Removed
+
+### Fixed
+
+### Security
+
+## [1.2.2] - 2024-12-13
+
+### Fixed
+
+- Fixed the galois_group problem.
```

Comment:
```suggestion
- Fixed the `galois_group` problem.
```
That's too vague and also the language is problematic (it sounds as if there was and is a unique problem, which is now solved, so the code is now perfect. No offense intended to anyone but that seems unlikely). Marginally better:
```suggestion
- Fixed a problem with `galois_group`.
```
But really, what was the problem? Since no PRs or issues are linked one can't easily find out :-(